docs/src/examples/components/Chat/Types/ChatMessageExampleStyled.shorthand.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { Chat, Provider } from '@stardust-ui/react'
+
+const janeAvatar = {
+  image: 'public/images/avatar/small/ade.jpg',
+  status: { color: 'green', icon: 'check' },
+}
+
+const content = (
+  <div>
+    Sure! Try one of these places:<br />
+    <a href="#">www.goodFood1.com</a>,<br />
+    <a href="#">www.goodFood2.com</a> or<br />
+    <a href="#">www.goodFood3.com</a>
+  </div>
+)
+
+const slotLabelStyles: any = (label, beforeStyles) => ({
+  position: 'relative',
+  border: '1px solid #000',
+  padding: '12px',
+  ':before': {
+    content: `'${label}'`,
+    position: 'absolute',
+    background: '#000',
+    paddingBottom: '2px',
+    bottom: '-1px',
+    right: '-1px',
+    color: 'white',
+    fontSize: '11px',
+    letterSpacing: '0.1px',
+    lineHeight: '9px',
+    ...beforeStyles,
+  },
+})
+
+const ChatMessageExampleStyled = () => (
+  <Provider
+    theme={{
+      componentStyles: {
+        ChatMessage: {
+          root: { ...slotLabelStyles('root'), backgroundColor: '#2E8B57' },
+          avatar: {
+            ...slotLabelStyles('avatar', { bottom: '-11px' }),
+            backgroundColor: '#FF00FF',
+            padding: 0,
+          },
+          messageBody: { ...slotLabelStyles('messageBody'), backgroundColor: '#87CEFA' },
```

Comment:
would be great to generalize this functionality and export under 'debug' prop to the client (or via any other generalized approach), in the same way it is done for Layout - that will reduce amount of code necessary in examples

Reply:
good proposals, let's discuss

Reply:
discussed with @kuzhelov and decided to leave this in examples so that users can understand how we achieved the custom styling